examples/bazel/create_pod.c

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
Do we need to have separate examples for bazel? Can we just use the existing example code in `create_pod/main.c` for example?

Reply:
good point @brendandburns , will take care of that.
Thanks!